source/documentation/other-topics/access-cross-aws-resources.html.md.erb

```diff
@@ -0,0 +1,259 @@
+
+1. **Create IAM role**
+ 
+    Cloud Platform maintain these cross account roles in [cloud-platform-infrastructure](https://github.com/ministryofjustice/cloud-platform-infrastructure/tree/master/terraform/cross-account-IAM) repo.
+
+    Create a file (<namespace>.tf), update the below template with correct values and raise a PR.
+
+    <details><summary> Click to see a template code block </summary>
+    <p>
+
+    ```
+    data "aws_iam_policy_document" "<namespace>-kiam-trust-chain" {
+      # KIAM trust chain to allow pods to assume roles defined below
+      statement {
+        principals {
+          type        = "AWS"
+          identifiers = [data.aws_iam_role.nodes.arn]
+        }
+        actions = ["sts:AssumeRole"]
+      }
+    }
+
+    variable "<namespace>-tags" {
+      type = map(string)
+      default = {
+        business-unit          = "<Which part of the MoJ is responsible for this service? (e.g HMPPS, Legal Aid Agency)>"
+        application            = "<Application name>"
+        is-production          = "<true/false>"
+        environment-name       = "<dev/test/staging/prod>"
+        owner                  = "<team responsible for this application>"
+        infrastructure-support = "<Email address for contact/support>"
+      }
+    }
+
+    resource "aws_iam_role" "<namespace>-<role-name>" {
+      name               = "<namespace>-<role-name>"
+      description        = "IAM role for XXXX"
+      tags               = var.<namespace>-tags
+      assume_role_policy = data.aws_iam_policy_document.<namespace>-kiam-trust-chain.json
+    }
+
+    resource "kubernetes_secret" "<secret-name-in-your-namespace>" {
+      metadata {
+        name      = "<secret-name-in-your-namespace>"
+        namespace = "<namespace>"
+      }
+
+      data = {
+        arn       = aws_iam_role.<namespace>-<role-name>.arn
+        name      = aws_iam_role.<namespace>-<role-name>.name
+        unique_id = aws_iam_role.<namespace>-<role-name>.unique_id
+      }
+    }
+
+    data "aws_iam_policy_document" "<namespace>-<role-name>" {
+
+      # allow pods to assume this role
+      statement {
+        actions   = ["sts:AssumeRole"]
+        resources = [aws_iam_role.<namespace>-<role-name>.arn]
+      }
+
+      # Provide list of permissions and target AWS account resources to allow access from
+      statement {
+        actions = [
+          "<some action>",
+        ]
+        resources = [
+          "<ARN of resource in target AWS account>/*",
+        ]
+      }
+    }
+
+    resource "aws_iam_policy" "<namespace>-<policy-name>" {
+      name   = "<namespace>-<policy-name>"
+      policy = data.aws_iam_policy_document.<namespace>-<policy-name>.json
+    }
+
+    resource "aws_iam_role_policy_attachment" "<namespace>-<policy-name>" {
+      role       = aws_iam_role.<namespace>-<role-name>.name
+      policy_arn = aws_iam_policy.<namespace>-<policy-name>.arn
+    }
+    ```
+    </p>
+    </details>
+
```

Comment:
Add something here like: 

Here is an example of terraform code based on the template above, with typical values provided.